bin/gcs_cli.py

```diff
@@ -0,0 +1,136 @@
+#!/usr/bin/env python
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+# Manipulate emulated GCS storage.
+
+# Usage: ./bin/gcs_cli.py CMD
+
+import os
+
+import click
+import requests
+import urllib3
+
+from google.auth.credentials import AnonymousCredentials
+from google.cloud import storage
+from google.cloud.exceptions import NotFound
+
+base_url = os.environ["STORAGE_EMULATOR_HOST"]
```

Comment:
Is there any reason we can't name this more specifically as `GCS_STORAGE_EMULATOR_HOST`?

Reply:
I think it needs to be STORAGE_EMULATOR_HOST to be picked up by the google cloud storage sdk, seeing as how this value is only used for printing messages